urifind.go

```diff
@@ -183,7 +179,8 @@ func PDFTitleGet(url string) string {
 		}
 	}()
 
-	gTitle, _, gErr := TitleGet("https://webcache.googleusercontent.com/search?q=cache:" + url)
+	c := http.Client{Timeout: 10 * time.Second}
```

Comment:
Weil TitleGet jetzt, damit man es besser testen kann, einen Doer verlangt.

Den http-client anzulegen ist keine teure Sache, schadet daher also nicht. In einem späteren commit wird das ohnehin nochmal refactored — PDFTitleGet sollte auch contexts benutzen, statt einem HTTP client mit Timeout.